src/file.rs

```diff
@@ -25,6 +25,15 @@ pub struct File<'a, IO: ReadWriteSeek, TP, OCC> {
     fs: &'a FileSystem<IO, TP, OCC>,
 }
 
+/// An extent containing a file's data on disk.
+///
+/// This is created by the `extents` method on `File`. All values in bytes.
```

Comment:
Missing verb: All values ARE in bytes.
Perhaps more detailed description what you mean by an extent would be a good idea like in `extents` fun doc (the regions on-disk occupied by this file.). Perhaps changing it to "byte ranges" would be better? Regions reminds me of winapi regions which are something very different